lib/src/main/java/io/ably/lib/rest/Auth.java

```diff
@@ -409,6 +403,21 @@ public JsonObject asJSON() {
 	 * @param options
 	 */
 	public TokenDetails authorize(TokenParams params, AuthOptions options) throws AblyException {
+		return authorize(params, options, true);
+	}
+
+	/**
+	 * Alias of authorize() (0.9 RSA10l)
+	 */
+	public TokenDetails authorise(TokenParams params, AuthOptions options) throws AblyException {
+		Log.w(TAG, "authorise() alias will be removed in 1.0");
```

Comment:
Can this be more explicit i.e. `authorise() is deprecated and will be removed in 1.0. Please use authorize() instead`?